fabbwled-backend/src/main/resources/sections/613-648.yaml

```
@@ -0,0 +1,370 @@
+        of Tyrnai. Subtract 1 from your COMBAT ability until you can
+        find a way of lifting the curse. Perhaps the priests of Sig in
+        Marlock City can help you. Note that you are under ‘Tyrnai’s
+        Curse, -1 COMBAT. You sneak out of the temple. Turn to 400 .
+    - turnTo: 400
+# - number: 626
+#   content:
+#     - text: |
+#         You recite a devotional prayer to the gods, calling on your faith
+#         to aid you. The goblin folk are repelled by your piety, and
+#         purity of spirit. Even Gobrash finds the strength to get out from
+#         under you, and run off, such is your effect on them.
+#         ‘Eeaurgh!’ snarls the queen, recoiling, ‘Please, your godliness
+#         is harmful to us. Here, take this and leave!’
+#         A jug of faery mead and an enchanted spear (COMBAT
+#         +2) are pushed towards you. Note them on your Adventure
+#         Sheet.
+#         The wall behind you shimmers and disappears – you step
+#         through, back into the cold, night air of the Curstmoor. You
+#         leave the faery mound far behind, and camp for the night. The
+#         next day, you resume your travels.
+#     - choice:
+#         - text: Go north across country turn to 560
+#           then:
+#             - turnTo: 560
+#         - text: Head east to the road turn to 558
+#           then:
+#             - turnTo: 558
+#         - text: Go to Trefoille turn to 250
+#           then:
+#             - turnTo: 250
+#         - text: Go to Marlock City turn to 100
+#           then:
+#             - turnTo: 100
+#         - text: Head west towards the River Grimm turn to 99
+#           then:
+#             - turnTo: 99
+- number: 627
+  content:
+    - text: |
+        A bard in the tavern recites an epic tale of ancient legend. He
+        speaks of Xinoc the Priest King, the ruler of a mighty nation in
+        the distant steppes, far to the north. A thousand years ago his
+        empire flourished, and his wealth was enormous. He was buried
+        in a great pyramid tomb, along with his riches. It is said the
+        Pyramid of Xinoc lies beyond the Spine of Harkun, and beyond
+        the Great Steppes, in the foothills of the Peaks at the Edge of the
+        World. You question him further, but that is all he knows. You
+        leave the tavern. Turn to 10
+    - turnTo: 10
+# Section 628 - too complex for the current implementation of sections
+# Section 629 - too complex for the current implementation of sections
+- number: 630
+  content:
+    - text: |
+        You struggle deeper into the forest until you come to a thick
+        wall of impenetrable thorn bushes. Circling it, you find there is a
+        break in the hedge, but it is filled by a large tree.
+        To your surprise, a face forms in the trunk, and speaks in a
+        woody voice, ‘None can pass – begone, human!’
+        If you have the codeword Apple , turn to 594 . If not, you
```

```suggestion
        If you have the codeword Apple , turn to 594 . If not, you can:
```

Maybe add can: here?